safeeyes/config/locale/de/LC_MESSAGES/safeeyes.po

```diff
@@ -549,7 +553,8 @@ msgstr ""
 # plugin/limitconsecutiveskipping
 #, python-format
 msgid "Skipped or postponed %(num)d/%(allowed)d breaks in a row"
-msgstr "%(num)d/%(allowed)d Pausen hintereinander übersprungen oder aufgeschoben"
+msgstr ""
+"%(num)d/%(allowed)d Pausen hintereinander übersprungen oder aufgeschoben"
```

Comment:
@deltragon I ran update-po.sh, and this automatically happened. I am unsure why

Reply:
Yeah, this is just formatting.